addOns/grpc/src/test/java/org/zaproxy/addon/grpc/internal/ProtoBufMessageDecoderUnitTest.java

```diff
@@ -131,6 +131,6 @@ void shouldDecodeWithCorruptedInput() {
                 "Failed to decode protobuf message: The message format is invalid or corrupted";
         assertEquals(expectedExceptionMessage, exception.getMessage());
         assertEquals(expectedOutput, decoder.getDecodedOuput());
-        assertEquals(0, decoder.getDecodedToList().size());
+        assertEquals(2, decoder.getDecodedToList().size());
```

Comment:
Should we assert the contents as well?

Reply:
maybe for corruption case we should not assert the content, as it can be different

Reply:
yes, actually for this case, the error is InvalidNumberFormat

Reply:
Bump.

Reply:
It should assert the content.

Reply:
but i am asserting in already in above line ,sorry , which content you are asking about

Reply:
The contents in the list.

Reply:
Ideally we should verify in all cases, but leave this for other PR as they are pre-existing tests.

Reply:
ok for now i will only fix the testcases name